bulk-ingester/src/main/java/org/alfresco/hxi_connector/bulk_ingester/processor/mapper/AlfrescoNodeMapper.java

```diff
@@ -80,6 +88,9 @@ public IngestEvent map(AlfrescoNode alfrescoNode)
         }
         allProperties.put(CREATED_AT_PROPERTY, createdAt);
 
+        allProperties.put(ALLOW_ACCESS, getResourceReaderAuthorities(alfrescoNode));
+        allProperties.put(DENY_ACCESS, getResourceDeniedAuthorities(alfrescoNode));
```

Comment:
I think it would be better to omit these fields for community installations. In that way it is really obvious if the document:
* Has ACLs set
* Has no ACLs set
* Does not specify whether there are ACLs set or not

Reply:
Ah - I just realised that this is for the Bulk Ingester, which uses the ES library and so always includes ACL information. In that case let's leave this as is.